test/unit/controllers/CustomsNotificationControllerSpec.scala

```diff
@@ -97,34 +96,34 @@ class CustomsNotificationControllerSpec extends UnitSpec with Matchers with Mock
         status(result) shouldBe ACCEPTED
       }
       
-      verify(mockCustomsNotificationService).handleNotification(meq(ValidXML), meq(expectedRequestMetaData), meq(apiSubscriptionFields))(any[HeaderCarrier]())
+      verify(mockCustomsNotificationService).handleNotification(meq(ValidXML), meq(expectedRequestMetaData), meq(apiSubscriptionFields))(any())
       verifyLog("info","Saved notification", mockNotificationLogger)
     }
 
     "respond with status 202 for missing Authorization when auth token is not configured" in {
       returnMockedCallbackDetailsForTheClientIdInRequest()
       when(mockConfigService.maybeBasicAuthToken).thenReturn(None)
-      when(mockCustomsNotificationService.handleNotification(meq(ValidXML), meq(expectedRequestMetaData.copy(maybeBadgeId = None, maybeSubmitterNumber = None)), meq(apiSubscriptionFields))(any[HeaderCarrier]())).thenReturn(eventualTrue)
```

Comment:
is this failing if we keep any[HeaderCarrier] or hc is not used there anymore?

Reply:
Not its not failing but we are getting rid of parametrised mocks following the recent library upgrades if I am right.